src/Microsoft.AspNet.Mvc.Core/ModelBinding/ByteArrayModelBinder.cs

```diff
@@ -39,7 +39,14 @@ public class ByteArrayModelBinder : IModelBinder
             try
             {
                 var model = Convert.FromBase64String(value);
-                return new ModelBindingResult(model, bindingContext.ModelName, isModelSet: true);
+                return new ModelBindingResult(
+                    model,
+                    bindingContext.ModelName,
+                    isModelSet: true,
+                    validationNode: new ModelValidationNode(
+                        bindingContext.ModelName,
+                        bindingContext.ModelMetadata,
+                        model));
```

Comment:
Also why does this model binder need to create its own `ModelValidationNode`?  When we had `ModelValidationNode` in the system before, only `CollectionModelBinder`, `MutableObjectModelBinder` and a few other special cases dealt with `ModelValidationNode` directly.